.github/workflows/ci.yml

```diff
@@ -24,7 +24,6 @@ jobs:
         include:
           - ros_distribution: melodic
           - ros_distribution: noetic
-          - ros_distribution: galactic
```

Comment:
comment from @jtbandes: We may want to keep this to prevent accidentally breaking compilation on galactic